data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -57,7 +57,7 @@ def prepare_census_data(workforce_census_path, pupil_census_path):
 
     school_pupil_census = pd.read_csv(
         pupil_census_path,
-        encoding="utf8",
+        encoding='cp1252',
         index_col=input_schemas.pupil_census_index_col,
```

Comment:
nitpick: needs a `make lint`…?

Reply:
Yep, not ran make lint on this yet... it is a big enough PR as is :)